Runtimes/Unity/BepInEx.Unity.IL2CPP/Preloader.cs

```diff
@@ -43,6 +42,8 @@ public static void Run()
             RedirectStdErrFix.Apply();
 
             ChainloaderLogHelper.PrintLogInfo(Log);
+            
+            UnityInfo.Initialize(Paths.ExecutablePath, Paths.GameDataPath);
 
```

Comment:
Why is delaying the initialization necessary?

Reply:
To enable the error message be logged. Until the console is initialized the error log is useless.